lib/cisco_node_utils/bgp_af.rb

```diff
@@ -98,21 +98,41 @@ def set_args_keys(hash={}) # rubocop:disable Style/AccessorMethodName
     #                      PROPERTIES                      #
     ########################################################
 
+    # IOS XR?
+    def ios_xr?
```

Comment:
If you don't do this in this commit - can you add a bug to our backlog to track this please?